docker/demoserver.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM learningequality/kolibribase
+
+ENV KOLIBRI_RUN_MODE=demoserver
+ENV KOLIBRI_PROVISIONDEVICE_FACILITY="Kolibri Demo"
+ENV WHICH_PYTHON=python2
```

Comment:
Python 2?

Reply:
The dockerbaseimage contains both python2 and python3
https://github.com/ivanistheone/kolibri/blob/dockerfiles3/docker/base.dockerfile#L15
and you can run latest Kolibri using either. The entrypoint.py script itself can run py2 or py3 as well.

In all scripts, `python2` is the default but users can set `WHICH_PYTHON=python3` to try py3.

Reply:
Should we use Python 3 as default then? :)

Reply:
So I played with both, and noticed sometimes the `develop` branch has trouble with py3 support (currently works fine, but with `kolibri-0.11.0a1.dev_git-37-gb15b923` there was some minor issue):

```
Importing content channels...
INFO     Option RUN_MODE in section [Deployment] being overridden by environment variable KOLIBRI_RUN_MODE
INFO     Option HTTP_PORT in section [Deployment] being overridden by environment variable KOLIBRI_HTTP_PORT
INFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base
Traceback (most recent call last):
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 365, in execute
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 293, in _wrap_coverage
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 325, in _wrap_profiling
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 408, in _execute
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 466, in execute_entry
  File "/kolibrihome/kolibri-0.11.0a1.dev_git-37-gb15b923.pex/.bootstrap/_pex/pex.py", line 471, in execute_module
  File "/usr/lib/python3.5/runpy.py", line 199, in run_module
...
  File "/root/.pex/install/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl.61bf88447e2c39f64018e226231282c2ecd52e1e/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl/kolibri/content/api_urls.py", line 4, in <module>
    from .api import (ChannelMetadataViewSet, ContentNodeGranularViewset,
  File "/root/.pex/install/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl.61bf88447e2c39f64018e226231282c2ecd52e1e/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl/kolibri/content/api.py", line 35, in <module>
    from kolibri.content.utils.stopwords import stopwords_set
  File "/root/.pex/install/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl.61bf88447e2c39f64018e226231282c2ecd52e1e/kolibri-0.11.0a1.dev0+git.37.gb15b923-py2.py3-none-any.whl/kolibri/content/utils/stopwords.py", line 8, in <module>
    stopwords = json.load(f)
  File "/usr/lib/python3.5/json/__init__.py", line 265, in load
    return loads(fp.read(),
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 103: ordinal not in range(128)
```

I vote for py2 as default because most devs use py2 for their dev environemt, though using py3 might be a good way to ensure py3 breakage is caught sooner.

Reply:
That error is probably fixed in this PR, should go away if you merge latest develop: https://github.com/learningequality/kolibri/pull/3769

I think @aronasorman has different thoughts about the demo Docker file, using it for demo server deployment :)

Anyways, no problem just doing Python 2 all around. Ensures we don't forget it.

Python 2 is often used for development because building the source distribution (`make dist` etc) requires Python 2.